drivers/platform/maxim/maxim_spi.c

```diff
@@ -56,6 +57,121 @@
 /************************ Functions Definitions *******************************/
 /******************************************************************************/
 
+/**
+ * @brief Enable the chip select gpio on MAX32665 or MAX32650
+ * @param spi - The SPI port registers
+ * @param chip_select - cs index
+ * @return 0 in case of success, -EINVAL otherwise
+ */
+static int32_t _max_spi_enable_ss(uint32_t id, uint32_t chip_select)
+{
+	mxc_gpio_cfg_t cs;
+
+	/** This function will be a no-op for other chips, because this step is
+	 * done in the SDK in those cases.
+	*/
+#if TARGET_NUM != 32650 && TARGET_NUM != 32665
+	return 0;
+#endif
+
+	switch(id) {
+	case 0:
```

Comment:
maybe we can use an enum with appropriate namings for these ids.

Reply:
In this case, the `id` specifies the index of the SPI port (the `device_id` of a SPI descriptor), so I don't think that there is a point in creating a define for it.